lua/refactoring/tasks/post_refactor.lua

```diff
@@ -9,6 +10,7 @@ local function create_post_refactor_tasks()
         :from_task(apply_text_edits)
         :add_task(format)
         :add_task(adjust_cursor)
+        :add_task(indent)
```

Comment:
i really feel like the indentation should be decided at extraction time and the text itself mutated instead of post applied.  I really think this will be hard to get right and furthermore, I think it will be literally impossible for python.